lib/trader.js

```diff
@@ -62,6 +62,11 @@ Trader.prototype.syncLogs = function syncLogs () {
   // Get last seen timestamp from server
   this.request({ path: '/logs', method: 'get' })
     .then(data => data.body)
+  // Delete log files that are two or more days old
+    .then(() => {
```

Comment:
data.body was passing through to logs.queryNewestLogs and now we return whatever logs.removeLogFiles return.